src/monitoring_service/cli.py

```diff
@@ -32,14 +37,15 @@
     "--confirmations",
     default=DEFAULT_NUMBER_OF_BLOCK_CONFIRMATIONS,
     type=click.IntRange(min=0),
-    help="Number of block confirmations to wait for",
+    help="Number of block confirmations to wait for",  # pylint: disable=too-many-arguments
```

Comment:
Where does that pylint exception come from? The code is the same as before and four args don't seem too many.

Reply:
I added an argument seven lines below, but `pylint` complained about this line.

Reply:
`raiden` client uses two `contracts_version`s: `PRODUCTION_CONTRACT_VERSION` and `DEVELOPMENT_CONTRACT_VERSION`  https://github.com/raiden-network/raiden/blob/93081972c11dc89c32ea0f2bab4f104baccf946e/raiden/settings.py#L46 .

Currently these two coincide, but sometimes they don't. Depending on the development/production flag given to `raiden` command, our codebase will be talking with different versions of contracts.

Reply:
I don't think we need such a split for the services, but maybe I'm missing something. @palango, what do you think?

Reply:
Good question. So fare we're always on `develop` but that will change once we release. I guess we need to see how the client will handle this going forward

Reply:
What for now? I'm happy with either way.

Reply:
Let's just keep it as is, and we fix it once it's clear how the client will handle it.

Reply:
Even if the client has some split between production and development, I don't see a need to support different contract versions inside a single release. Thus I suggest replacing this PR by https://github.com/raiden-network/raiden-services/pull/417.